src/pages/Home.jsx

```diff
@@ -3,42 +3,16 @@ import axios from 'axios'
 import classNames from 'classnames'
 import React from 'react'
 import { useQuery } from 'react-query'
+import { Article, PopularTags } from '../components'
+import { useArticles } from '../hooks'
 
 const DEFAULT_LIMIT = 10
 
 function Home() {
-  const [offset, setOffset] = React.useState(0)
-  const [activeTag, setActiveTag] = React.useState(null)
-  const tagsQuery = useQuery('get-tags', () => axios.get('https://conduit.productionready.io/api/tags'), {
-    placeholderData: {
-      data: {
-        tags: [],
-      },
-    },
-  })
-  const articlesQuery = useQuery(
-    ['get-article-list', { offset, tag: activeTag }],
-    () =>
-      axios.get('https://conduit.productionready.io/api/articles', {
-        params: {
-          limit: DEFAULT_LIMIT,
-          offset,
-          tag: activeTag,
-        },
-      }),
-    {
-      placeholderData: {
-        data: {
-          articles: [],
-          articlesCount: 0,
-        },
-      },
-    }
-  )
-
-  const articles = articlesQuery?.data?.data?.articles
-  const articlesCount = articlesQuery?.data?.data?.articlesCount
-  const tags = tagsQuery?.data?.data?.tags
+  const [filters, setFilters] = React.useState({offset:0, limit:DEFAULT_LIMIT, tag: null})  
```

Comment:
I don't think you want to colocate all of the "filters" together here... The limit never changes so there is no reason to keep it in state, the offset changes only when you click on the pagination and the tag only changes when clicking on the tags / tabs. So I think in this case it makes more sense to separate into two pieces of state.